src/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)
+set(BUILD_SHARED_LIBS ON)
```

Comment:
may it be wrapped by an `if` check?

Reply:
What to calibrate?😰

If you want to generate `.lib` manually change it to **OFF**.